### PR TITLE
Happychat - add explicit label in dev env to chat button

### PR DESCRIPTION
**Pull request by mkaz**
In dev environment you can not tell what chat service you will get when launching "Chat with us"
So to avoid launching unnecessary Olark chats, explicitly label the button with Happychat

### CODE REVIEW

File: client/me/help/help-contact/index.jsx [hunk 1]
```diff
@@ -481,7 +481,7 @@ const HelpContact = React.createClass( {
 			},
 			showHappychatVariation && {
 				onSubmit: this.startHappychat,
-				buttonLabel: this.translate( 'Chat with us' )
+				buttonLabel: ( config( 'env' ) === 'development' ) ? 'Happychat with us' : this.translate( 'Chat with us' )
```
Comment (omarjackman) [inline client/me/help/help-contact/index.jsx]: We should translate this

Comment (mkaz) [inline client/me/help/help-contact/index.jsx]: Happychat is Happychat in all languages, like Taxi.
Also it only shows in dev or staging environment which is most likely Automatticians

**Comment by mkaz** [comment]
Updated to include button label within staging environment

**Comment by georgeh** [comment]
LGTM, :shipit:

**Pull request merged**